### ISSUE TITLE
Unify Accessibility Checker behaviour when no issues are detected

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#5]
We should unify behavior  of Accessibility Checker when no a11y issues are detected.
### Checking empty document
1. Open any sample with Accessibility Checker. (i.e. [plugins/a11ychecker/dev/sample.html](http://ckeditor.dev/plugins/a11ychecker/dev/sample.html))
2. Clear the CKEditor contents, using "New Page" button.
3. Execute Accessibility Checker, eg. by clicking a its toolbar button.
### Fixing the last problematic element
1. Open any sample with Accessibility Checker. (i.e. [plugins/a11ychecker/dev/sample.html](http://ckeditor.dev/plugins/a11ychecker/dev/sample.html))
2. Set the CKEditor contents using "Source" button to:

``` html
<p>This is a sample date: 12-04-12.</p>
```
1. Execute Accessibility Checker.
2. Navigate to the issue, eg `CTRL + E`.
3. Apply a Quick Fix.

**Expected result:**
For the time being alert with an appropriate message will be sufficient.
